lms/templates/courseware/progress.html

```diff
@@ -45,7 +45,8 @@
       % endif
 
       <header class="progress-certificates">
-        <h1 class="progress-certificates-title">${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email)}</h1>
+        ## Translators: Edraak-specific
+        <h1 class="progress-certificates-title">${_(u"Course Progress for Student '{username}' \u200f ({email})").format(username=student.username, email=student.email)}</h1>
```

Comment:
This should be in Transifex and only for the Arabic language, unless there's a bug in the English interface.